packages/build/src/plugins/child/track.js

```diff
@@ -41,24 +41,6 @@ const trackObjectMutations = function (value, keys, { configMutations, event })
   return value
 }
 
-// Inverse of `trackObjectMutations()`
-const removeProxies = function (value) {
```

Comment:
This is not needed anymore since the `value` is now JSON serialized when passed from the child to the parent process, which removes any `Proxy`.